tests/BUILD

```diff
@@ -28,6 +28,16 @@ cris_cc_test (
     flaky = True,
 )
 
+cris_cc_test (
+    name = "recorder_snapshot_test",
+    srcs = ["recorder_snapshot_test.cc"],
+    deps = [
+        "//:msg_recorder",
+        "@gtest//:gtest_main",
+    ],
+    flaky = True,
```

Comment:
why? it should not be flaky

Reply:
We only mark something as flaky when we don't know how to fix it while still believe it's OK to leave it as-is for a while, or we think a test has a variance way larger than the tolerance by-design without a good way to reduce it.

All flaky-by-design tests should have strong justification.